docs(backend): Reflect latest changes in authorization for SQL API (#3663)
cube-js/cube.js: docs/content/Cube.js-Backend/SQL-API.mdx

</InfoBox>

<DangerBox>

> Before the v0.28.55 version, Cube SQL used the same JWTs as Cube, but we moved to credentials authentication as a workaround of username

Reviewer: ```suggestion
Prior to the v0.28.55 version, Cube SQL used the same JWTs as Cube, but we moved to credentials authentication as a workaround for username
```

Author: Thanks!
